cluster/charts/crossplane-controllers/.gitignore

```diff
@@ -0,0 +1,2 @@
+values.yaml
+values.yaml-e
```

Comment:
This would appear to be a typo'd filename.  Is it?

Reply:
No, this file is generated after a `make build`
Same `.gitignore` already exist for current helm chart: https://github.com/crossplaneio/crossplane/blob/master/cluster/charts/crossplane/.gitignore